src/content/ccip/concepts/manual-execution.mdx

```diff
@@ -0,0 +1,115 @@
+
+When manual execution is initiated, a Merkle proof is generated for the message to be executed. During execution, The CCIP explorer submits the Merkle proof and the new gas limit (if the initial failure was due to a low gas limit). This Merkle proof is verified by the [OffRamp contract](/ccip/architecture#offramp) against the Merkle root in the [CommitStore contract](/ccip/architecture#commit-store), and that was blessed by the [Risk Management Network](/ccip/architecture#risk-management-network). This mirrors the automated execution performed by the [CCIP Executing DON](/ccip/architecture#executing-don), with the addition that the user-provided gas limit is used for the resubmission.
+
+## Frequently Asked Questions
+
+1. **Can anyone execute a transaction on the CCIP explorer even if they are not the initiator of the transaction?**
+
+   Yes, any EOA can manually execute a CCIP message that is eligible for manual execution. However, the executing account must have sufficient native gas tokens (such as ETH on Ethereum or MATIC on Polygon) to cover the gas costs associated with the delivery of the CCIP message.
+
+   <br />
+
+1. **If a user sends multiple messages and the first message isn’t successfully delivered and goes into a _manual execution_ mode, does that mean all subsequent messages from the user will also be stuck?**
+
+   It depends. If a message goes into manual execution mode due to receiver errors (unhandled exceptions or gas limit issues), subsequent messages don’t get automatically blocked, unless they would encounter the same error. However, suppose a message goes into manual execution mode after the Smart Execution time window expires (currently 8 hrs). In that case, subsequent messages must wait for the first message to be processed to maintain the default sequence.
+
+   <br />
+
+1. **If the maximum gas limit is 2M (2,000,000) on mainnet, but it turns out that the destination blockchain requires more than that, will an override of > 2M work even though the max gas limit is 2M?**
+
+   Yes, but only for this execution. This works because the gas limit for this execution instance isn’t passing through the CCIP validation for the gas limit, for which the CCIP executing DON pays the gas. However, if you consistently need more than 2M for your use case, please reach out to us via <CopyText text="support@chain.link" /> and let us know.
+
+   <br />
+
+1. **Will Chainlink Labs reimburse us for manual execution fees?**
+
+   Since most manual execution situations are due to insufficient gas limit or an unhandled exception in the receiver contract, Chainlink Labs does not reimburse these fees. If you are a dApp developer, please ensure you test thoroughly to avoid manual execs to the extent possible.
```

Comment:
```suggestion
   Since most manual execution situations are due to insufficient gas limit or an unhandled exception in the receiver contract, Chainlink Labs does not reimburse these fees. If you are a dApp developer, please ensure you test thoroughly to avoid manual executions to the extent possible.
```